packages/jazz-tools/src/tools/coValues/CoValueBase.ts

```diff
@@ -21,29 +20,59 @@ import type {
 } from "../internal.js";
 
 /** @internal */
+export abstract class CoValueBase implements CoValue {
+  declare $type: string;
 
-export class CoValueBase implements CoValue {
-  declare id: ID<this>;
-  declare _type: string;
-  declare _raw: RawCoValue;
+  declare abstract $jazz: CoValueJazzApi<this>;
```

Comment:
Since the whoule point of `$jazz` is to reduce conflicts, how about giving to this a more internal name?

Something like `$type$` or `$$type`
This is meant for internal usage and hacks anyway, so it's ok to keep it ugly

Reply:
Done! Renamed it to `$type$`, to align it with other fields like `$items$` and `$SchemaInit$`